health/micro-ui-react19/web/packages/modules/campaign-manager/src/utils/setupCampaignValidators.js

```
@@ -381,7 +385,9 @@ export const  handleValidate = ({formData,t,setShowToast,hierarchyDefinition,low
       return true;
 
     case "deliveryRule":
-      const deliveryRules = formData?.deliveryRule;
+      // Tolerate an absent/malformed value here (this data can arrive via totalFormData instead) -
+      // the payload builder is what guards against overwriting saved rules
+      const deliveryRules = Array.isArray(formData?.deliveryRule) ? formData.deliveryRule : [];
```

Biome (2.5.6)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)